examples/service-fabric/windows-vmss-self-signed-certs/3-servicefabric.tf

```diff
@@ -0,0 +1,112 @@
+
+resource "azurerm_service_fabric_cluster" "example" {
+  name                = "exampleservicefabric"
```

Comment:
other examples use the variables `prefix` and `location` so users don't conflict with global resources - could we use those here too